your-rents-services-geodata/src/main/java/com/yourrents/services/geodata/model/Country.java

```diff
@@ -0,0 +1,15 @@
+package com.yourrents.services.geodata.model;
+
+import java.util.UUID;
+
+public record Country(String isoCode, String localName, String englishFullName, UUID uuid) {
```

Comment:
Some data are missed: iso3 code, number code and the reference to the continent.

Reply:
yes it was just a partial implementation. I will add these fields

Reply:
Why do you need the CRUD for continent, for having am uuid reference to the continent?